### PR TITLE
add usage example for ```azurerm_monitor_diagnostic_setting``` integration with ```eventhub```

### LINKED ISSUE #12240: Add usage example for azurerm_monitor_diagnostic_setting integration with eventhub
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

Please add usage example for azurerm_monitor_diagnostic_setting integration with eventhub.

### New or Affected Resource(s)

* azurerm_monitor_diagnostic_setting

### PR DESCRIPTION
**Pull request by xiaxyi**
Added sample usage for the resource ```azurerm_monitor_diagnostic_setting```, as for the diagnostic settings, there are three destinations for users to choose from:
```
1. Log Analytics workspace
2. event hubs
3. azure storage account
```
added the example for the resource integration with the destination - eventhub

Fixes #12240

### CODE REVIEW

File: examples/azure-monitoring/eventhub_integration/variables.tf [hunk 1]
```diff
@@ -0,0 +1,7 @@
+variable "prefix" {
```
Comment (magodo) [inline examples/azure-monitoring/eventhub_integration/variables.tf:1]: As this is not used in your example, can we remove this file?

File: website/docs/r/monitor_diagnostic_setting.html.markdown [hunk 2]
```diff
@@ -63,8 +63,6 @@ The following arguments are supported:
 
 * `eventhub_name` - (Optional) Specifies the name of the Event Hub where Diagnostics Data should be sent. Changing this forces a new resource to be created.
 
--> **NOTE:** If this isn't specified then the default Event Hub will be used.
```
Comment (magodo) [inline website/docs/r/monitor_diagnostic_setting.html.markdown:66]: Can we revert this change on the document as this PR aims to add examples?

**Review by magodo — CHANGES_REQUESTED** [review]
Thanks for this PR!
I've taken a look through and left some comments inline, but this is mostly looking good to me 👍

**Comment by magodo** [comment]
@xiaxyi Thanks! LGTM :+1:

File: examples/azure-monitoring/eventhub_integration/variables.tf (hunk 1 above)
Comment (xiaxyi) [reply examples/azure-monitoring/eventhub_integration/variables.tf]: Thanks for the comments, I removed the file

File: website/docs/r/monitor_diagnostic_setting.html.markdown (hunk 2 above)
Comment (xiaxyi) [reply website/docs/r/monitor_diagnostic_setting.html.markdown]: added back the original comments
